sys/arm64/vmm/vmm_arm64.c

```diff
@@ -514,8 +516,11 @@ vmmops_init(struct vm *vm, pmap_t pmap)
 	vtimer_vminit(hyp);
 	vgic_vminit(hyp);
 
-	hyp->el2_addr = el2_map_enter((vm_offset_t)hyp, size,
-	    VM_PROT_READ | VM_PROT_WRITE);
+	prot = VM_PROT_READ | VM_PROT_WRITE;
+#if __has_feature(capabilities)
```

Comment:
Why are some of these unconditional and some conditional?

Reply:
As in, why am I using an ifdef here? I suppose there's no need to.